### PR TITLE
Add iOS event schema and MobileEventPing

### PR DESCRIPTION
**Pull request by akkomar**
This adds iOS Fennec/Amplitude integration

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  def getCreated: Option[Long] = Some(created)
+
+  override def pingAmplitudeProperties: JObject = {
+    ("device_id" -> getClientId) ~
+    ("device_model" -> device) ~
+    ("arch" -> arch) ~
+    ("locale" -> locale) ~
+    ("user_properties" ->
+      ("pref_defaultSearchEngine" -> settings.defaultSearchEngine) ~
```
Comment (fbertsch) [inline src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala]: Just want to point out that if it's easier for the PMs, you could make these pretty-printed names instead (e.g. `Default-Search-Engine` or similar). I'm not sure if spaces are allowed.

File: configs/fennec_ios_events_schemas.json [hunk 2]
```diff
@@ -0,0 +1,1800 @@
...
+                    "amplitudeProperties": {}, 
+                    "description": "", 
+                    "name": "Tapping Bookmarks button in New Tab Settings", 
+                    "schema": {
+                        "$schema": "http://json-schema.org/schema#", 
+                        "properties": {
+                            "category": {
+                                "enum": [
+                                    "action"
+                                ],
```
Comment (fbertsch) [inline configs/fennec_ios_events_schemas.json]: Is `OTHER` needed?

Comment (fbertsch) [inline configs/fennec_ios_events_schemas.json]: nit: `Tapped`? Here and elsewhere

Comment (akkomar) [reply configs/fennec_ios_events_schemas.json]: :+1:, it's not.

File: src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala (hunk 1 above)
Comment (akkomar) [reply src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala]: Hey @justindarc - would you like some nice, normalized property names here [1]?
I copied these 1:1 from your telemetry data as there seem to be couple of naming conventions in use, but maybe pretty-printing them as Frank suggests would make your life easier?

[1] https://github.com/mozilla/telemetry-streaming/blob/ios-amplitude/src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala#L42-L59

Comment (justindarc) [reply src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala]: Yes, if you could map them to pretty-printed labels, that would probably be easier for the PMs. Do you need me to provide you the labels or can you infer them from what's there?

**Pull request merged**